diff --color -Naur a/config.def.h b/config.def.h
--- a/config.def.h	2024-01-10 14:04:59.579655094 -0500
+++ b/config.def.h	2024-01-10 14:05:28.711844529 -0500
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Monospace:size=10:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
