diff --color -Naur a/Makefile b/Makefile
--- a/Makefile	2024-01-10 13:43:59.213820121 -0500
+++ b/Makefile	2024-01-10 13:44:18.455834004 -0500
@@ -48,8 +48,6 @@
 	mkdir -p $(DESTDIR)$(MANPREFIX)/man1
 	sed "s/VERSION/$(VERSION)/g" < st.1 > $(DESTDIR)$(MANPREFIX)/man1/st.1
 	chmod 644 $(DESTDIR)$(MANPREFIX)/man1/st.1
-	tic -sx st.info
-	@echo Please see the README file regarding the terminfo entry of st.
 	mkdir -p $(DESTDIR)$(APPPREFIX)
 	cp -f st.desktop $(DESTDIR)$(APPPREFIX)
 
