diff --color -Naur a/scripts/install-to-dest.sh b/scripts/install-to-dest.sh
--- a/scripts/install-to-dest.sh	2023-07-19 14:28:12.583077873 -0400
+++ b/scripts/install-to-dest.sh	2023-07-19 14:41:31.571802059 -0400
@@ -31,24 +31,22 @@
   --exclude '*' \
   --prune-empty-dirs \
   --delete
-chmod -R g-w,a-w src/ $PKGDIR/usr/share/yabsnap
 if $(which selinuxenabled 2>/dev/null); then
   restorecon -R src/ $PKGDIR/usr/share/yabsnap
 fi
 
 mkdir -p $PKGDIR/usr/bin
 ln -sf /usr/share/yabsnap/yabsnap.sh $PKGDIR/usr/bin/yabsnap
-chmod 755 $PKGDIR/usr/bin/yabsnap
+chmod 755 $PKGDIR/usr/share/yabsnap/yabsnap.sh
 
 mkdir -p $PKGDIR/usr/lib/systemd/system
-install -Z artifacts/services/yabsnap.service $PKGDIR/usr/lib/systemd/system
-install -Z artifacts/services/yabsnap.timer $PKGDIR/usr/lib/systemd/system
-sudo systemctl daemon-reload
+install -m0644 artifacts/services/yabsnap.service $PKGDIR/usr/lib/systemd/system
+install -m0644 artifacts/services/yabsnap.timer $PKGDIR/usr/lib/systemd/system
 
 readonly HOOKDIR=/usr/share/libalpm/hooks/
 if [[ -d "$HOOKDIR" ]]; then
   mkdir -p $PKGDIR/$HOOKDIR
-  install -Z artifacts/pacman/05-yabsnap-pacman-pre.hook $PKGDIR/$HOOKDIR
+  install -m0644 artifacts/pacman/05-yabsnap-pacman-pre.hook $PKGDIR/$HOOKDIR
 else
   printf 'Not an Arch based distro, will not install hook.\n' >&2
 fi
