diff --color -Naur a/src/code/snap_operator.py b/src/code/snap_operator.py
--- a/src/code/snap_operator.py	2023-07-26 23:33:08.293786038 -0400
+++ b/src/code/snap_operator.py	2023-07-26 23:43:08.716537659 -0400
@@ -103,7 +103,7 @@
             snapshot.create_from(self._config.source)
 
         # Clean up old snaps; leave count-1 previous snaps (plus the one now created).
-        for expired in previous_snaps[: -count + 1]:
+        for expired in previous_snaps[: len(previous_snaps) - (count - 1)]:
             expired.delete()
             self.need_sync = True
 
