diff --color -Naur a/config.h b/config.h
--- a/config.h	2024-05-17 15:59:18.549145300 -0400
+++ b/config.h	2024-05-17 16:03:57.118628663 -0400
@@ -359,8 +359,8 @@
 	{ XK_ANY_MOD,           Button2, selpaste,       {.i = 0},      1 },
 	#endif // CLIPBOARD_PATCH
 	#if SCROLLBACK_MOUSE_PATCH
-	{ ShiftMask,            Button4, kscrollup,      {.i = 1},      0, S_PRI},
-	{ ShiftMask,            Button5, kscrolldown,    {.i = 1},      0, S_PRI},
+	{ ShiftMask,            Button4, kscrollup,      {.i = 3},      0, S_PRI},
+	{ ShiftMask,            Button5, kscrolldown,    {.i = 3},      0, S_PRI},
 	#elif UNIVERSCROLL_PATCH
 	{ XK_ANY_MOD,           Button4, ttysend,        {.s = "\033[5;2~"}, 0, S_PRI },
 	{ XK_ANY_MOD,           Button5, ttysend,        {.s = "\033[6;2~"}, 0, S_PRI },
